backend/protocol_rpc/aio.py

```
@@ -0,0 +1,33 @@
+)
+
+import asyncio
+import concurrent.futures
+
+MAIN_SERVER_LOOP = asyncio.new_event_loop()
+MAIN_LOOP_EXITING = MAIN_SERVER_LOOP.create_future()
+
+MAIN_LOOP_DONE = concurrent.futures.Future()
+
+
+async def run_in_main_server_loop(initial_result):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)